jena-arq/src/main/java/org/apache/jena/sparql/lang/sparql_11/SPARQLParser11Constants.java

```diff
@@ -431,6 +433,7 @@
     "\"select\"",
     "\"distinct\"",
     "\"reduced\"",
+    "\"json\"",
```

Comment:
Oh, when updating the branch (it was written when Jena was 2.x) I simply applied the same changes to master.jj and sparql_11.jj, and looked at other classes that I had patched, and looked for similar classes that looked like had to be updated as well.

But maybe I added Should I revert this one line?